exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var roughStudent = Math.round(numberOfStudents)
+var roughMentor = Math.round(numberOfMentors)
+console.log(roughStudent)
+console.log(roughMentor)
+//I could not solve this problem 
```

Comment:
Math.round() rounds a decimal to the nearest whole number. you need to convert to percentages 

```
const total = numberOfMentors + numberOfStudents;
const percentageOfStudents = (numberOfStudents / total) * 100;
const percentageOfMentors = (numberOfMentors / total) * 100;
const roundedPercentageOfStudents = Math.round(percentageOfStudents);
const roundedPercentageOfMentors = Math.round(percentageOfMentors);
const message1 = "Percentage students: " + roundedPercentageOfStudents + "%";
const message2 = "Percentage mentors: " + roundedPercentageOfMentors + "%";
console.log(message1);
console.log(message2);
```